lib/transports.js

```diff
@@ -1,14 +1,16 @@
 var events = require('events');
 var util = require('util');
+var assign = require('object-assign');
```

Comment:
How is that different than just?

```javascript
for (var key in this.options) this.options.hasOwnProperty(key)) && (options[key] = this.options[key]);
```

Or maybe I'm missing something on what this does. But we're just simply merging the two objects.